src/ui/components/ProgressBar.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-    <div class="c-progress-bar">
-        <div class="c-progress-bar__holder">
-            <div class="c-progress-bar__bar"
-                 :class="{'--indeterminate': model.progressPerc === 'unknown'}"
-                 :style="`width: ${model.progressPerc}%;`">
-            </div>
-        </div>
-        <div class="c-progress-bar__text"
-             v-if="model.progressText !== undefined">
-            <span v-if="model.progressPerc > 0">{{model.progressPerc}}% complete.</span>
-            {{model.progressText}}
-        </div>
+<div class="c-progress-bar">
+    <div class="c-progress-bar__holder">
+        <div
+            class="c-progress-bar__bar"
+            :class="{'--indeterminate': model.progressPerc === 'unknown'}"
+            :style="`width: ${model.progressPerc}%;`"
```

Comment:
Using a JavaScript template string doesn't seem right here...Does it? How is this even working? Again, sorry, I know this is not your code 😄 Maybe we're lucky and either Vue is just ignoring the fact that it's a template string, or - less likely - the template is valid within the context that Vue is evaluating it.

Reply:
Looks like you _can_ use a template literal as per https://github.com/vuejs/vue/issues/2364. Still this is not recommended and should be provided by a computed property.

will fix

Reply:
fixed in 8b822b62025626151344dbb2f858ff83d237d842